### PR TITLE
Fix up the integration tests. 

### PR DESCRIPTION
**Pull request by haripate**
Resolving issues occurred by ISV-3309

### CODE REVIEW

File: operator-pipeline-images/Dockerfile [hunk 1]
```diff
@@ -61,6 +61,18 @@ WORKDIR /home/user
...
 
+# install PDM
+RUN pip3 install --upgrade pip setuptools wheel && pip3 install pdm
+# install dependencies in virtual environment
+COPY ./pdm.lock ./pyproject.toml ./README.md ./
+RUN pdm venv create 3.10 && pdm install --no-lock --no-editable \
+        --group operatorcert \
+        --group operatorcert-dev
+
+ENV VIRTUAL_ENV=./.venv
```
Comment (kosciCZ) [inline operator-pipeline-images/Dockerfile:72]: Perhaps there's some confusion about relative/absolute path here. In [pyxis](https://gitlab.cee.redhat.com/metadata/pyxis/pyxis/-/blob/master/dockerfiles/Dockerfile.graphql#L23), we use a absolute path, which 
```suggestion
ENV VIRTUAL_ENV=/home/user/.venv
```

File: operator-pipeline-images/Dockerfile [hunk 2]
```diff
@@ -61,6 +61,18 @@ WORKDIR /home/user
...
+RUN python3 -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+
 RUN pip3 install .
```
Comment (kosciCZ) [inline operator-pipeline-images/Dockerfile:76]: No longer needed if you're using pdm
```suggestion
```

**Review by kosciCZ — CHANGES_REQUESTED** [review]

File: operator-pipeline-images/Dockerfile [hunk 3]
```diff
@@ -61,6 +61,18 @@ WORKDIR /home/user
...
+# install PDM
+RUN pip3 install --upgrade pip setuptools wheel && pip3 install pdm
+# install dependencies in virtual environment
+COPY ./pdm.lock ./pyproject.toml ./README.md ./
+RUN pdm venv create 3.10 && pdm install --no-lock --no-editable \
+        --group operatorcert \
+        --group operatorcert-dev
+
+ENV VIRTUAL_ENV=./.venv
+RUN python3 -m venv $VIRTUAL_ENV
```
[the remark this answers is not in the document]
Comment (haripate) [reply operator-pipeline-images/Dockerfile]: Updated.

File: operator-pipeline-images/Dockerfile (hunk 2 above)
Comment (haripate) [reply operator-pipeline-images/Dockerfile]: Updated.

File: operator-pipeline-images/Dockerfile (hunk 1 above)
Comment (haripate) [reply operator-pipeline-images/Dockerfile]: Updated.

**Comment by haripate** [comment]
@Allda Can you please review the code changes?

File: tox.ini [hunk 4]
```diff
@@ -11,25 +11,24 @@ skipsdist = True
...
        --cov-fail-under 100
 
 [testenv:black]
-deps = -r operator-pipeline-images/requirements-dev.txt
+groups = operatorcert-dev
 commands = black --check --diff .
 
 [testenv:black-format]
-deps = -r operator-pipeline-images/requirements-dev.txt
+groups = operatorcert
```
Comment (Allda) [inline tox.ini:26]: Shouldn't this be `operatorcert-dev` instead?

**Review by Allda — APPROVED** [review]
One minor comment, but otherwise it looks good.

File: tox.ini (hunk 4 above)
Comment (haripate) [reply tox.ini]: Updated.

**Pull request merged**